packages/docs/base/color.md

```diff
@@ -1,57 +1,614 @@
-
-### Blue
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-blue-000"></li>
-  <li class="sample--color is-sample-blue-300"></li>
-  <li class="sample--color is-sample-blue-500"></li>
-  <li class="sample--color is-sample-blue-900"></li>
-</ul>
-
-### Green
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-green-000"></li>
-  <li class="sample--color is-sample-green-300"></li>
-  <li class="sample--color is-sample-green-500"></li>
-  <li class="sample--color is-sample-green-900"></li>
-</ul>
-
-### Red
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-red-000"></li>
-  <li class="sample--color is-sample-red-300"></li>
-  <li class="sample--color is-sample-red-500"></li>
-  <li class="sample--color is-sample-red-900"></li>
-</ul>
-
-### Yellow
-<ul class="sample--color-list">
-  <li class="sample--color
-   is-sample-yellow-000"></li>
-  <li class="sample--color is-sample-yellow-300"></li>
-  <li class="sample--color is-sample-yellow-500"></li>
-  <li class="sample--color is-sample-yellow-900"></li>
-</ul>
-
-
-## Secondary palette
-
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-purple-500"></li>
-  <li class="sample--color is-sample-turquoise-500"></li>
-  <li class="sample--color is-sample-orange-500"></li>
-</ul>
+Here's a quick overview of the colors available in Odyssey and some guidelines for their use.
+
+<figure class="ods-table--figure">
+  <figcaption class="ods-table--figcaption">
+    Neutral Palette
+  </figcaption>
+  <table class="ods-table">
+    <thead>
+      <tr>
+        <th scope="column">Swatch</th>
+        <th scope="column">Name</th>
+        <th scope="column">Hex</th>
+        <th scope="column">✅ Do</th>
+        <th scope="column">🚫 Don't Do</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td class="is-sample-table-swatch">
+          <span class="sample-color sample-color--gray-900"></span>
+        </td>
+        <td class="is-sample-table-token">
+          Gray 900
+        </td>
+        <td>
+          #212126
+        </td>
+        <td>
+          Titles, body copy, icons, shadows, tooltips
+        </td>
+        <td>
+          Background colors, buttons, strokes
+        </td>
+      </tr>
+      <tr>
+        <td class="is-sample-table-swatch">
+          <span class="sample-color sample-color--gray-600"></span>
+        </td>
+        <td class="is-sample-table-token">
+          Gray 600
+        </td>
+        <td>
+          #6e6e78
+        </td>
+        <td>
+          Default checkbox & radio borders
+        </td>
+        <td>
+          Titles, body copy, borders, icons
+        </td>
+      </tr>
+      <tr>
+        <td class="is-sample-table-swatch">
+          <span class="sample-color sample-color--gray-400"></span>
+        </td>
+        <td class="is-sample-table-token">
+          Gray 400
+        </td>
+        <td>
+          #aaaab4
+        </td>
+        <td>
+          N/A
```

Comment:
@arnoldsandoval-okta @joejezowski-okta @johnintrater-okta should we remove colors that have no approved use cases?

Reply:
Removed, resolved offline with @joejezowski-okta